guide/14-deep-learning/use_mistral_llm_for_text_classification_and_entity_recognition.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<p>Prepare databunch for the <code>TextClassifier</code> model using the <code>prepare_textdata</code> method in <code>arcgis.learn</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>